### ISSUE TITLE
1.20.1 server crashing

### ISSUE THREAD

**Issue by zer0cares** [62832/MEGACells#103]
[crash-2024-01-22_08.05.24-server.txt](https://github.com/62832/MEGACells/files/14008440/crash-2024-01-22_08.05.24-server.txt)

Worked fine before, then the only change was that the forge version on the server updated, and now the server doesn't start.  Tried reverting to the older forge version and its still crashing, with the crashlogs pointing to MEGACells.

Any help would be appreciated, thanks!

**Comment by xSamTheKingx** [comment]
Same issue here please help anyone?

**Comment by 62832** [comment]
Are we talking Forge or NeoForge?

**Comment by zer0cares** [comment]
Forge

**Comment by xSamTheKingx** [comment]
Forge_1.20.1_47.2.19

**Comment by 62832** [comment]
Then I would suggest using NeoForge instead since I'm not really providing support for Forge versions beyond 47.1.3.

**Comment by xSamTheKingx** [comment]
Its the server ram caused the issue. I upgraded ram to 16GB and now its good
Its strange how much ram atm9 needs as i only have 3 players on the server.

**Comment by 62832** [comment]
Yes, I suppose the initial caching process for the bulk compression service would be quite memory-intensive, though this certainly isn't ideal.
